### PR TITLE
Make a better dev experience

### LINKED ISSUE #147: Migrate from node-sass

### PR DESCRIPTION
**Pull request by TheJoin95**
# Pull Request

Closes #147 

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
package install & setup project locally

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* IGN Version:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/ign-frontend/src/views/Home.vue [hunk 1]
```diff
@@ -76,7 +76,7 @@
               <div class="circle"></div>
           </div>
           <div>
-            <strong>Total converted images: </strong>{{ apiCount }}
+            <strong>Total converted images: </strong>{{ apiCount.toString().replace(/[^\d]|[\']/g, '') }}
```
Comment (Wabri) [inline src/ign-frontend/src/views/Home.vue:79]:
```suggestion
            <strong>Total converted images: </strong>{{ apiCount.toString().replace(/^b'|'$/g, '') }}
```

even this solution have the same readability. Drop it if you dont like it.

File: src/ign-frontend/src/components/Demo.vue [hunk 2]
```diff
@@ -262,7 +262,7 @@ export default Vue.component('Demo', {
         });
         document.querySelector('.preview').classList.toggle('processing');
 
-        if (conversionCount !== 0 && conversionCount % 3 === 0) {
+        if (conversionCount === 0 || conversionCount % 3 === 0) {
```
Comment (Wabri) [inline src/ign-frontend/src/components/Demo.vue:265]: What was the problem here?

**Review by Wabri — COMMENTED** [review]
Some comments, and why this changes increase devex?

File: src/ign-frontend/src/components/Demo.vue (hunk 2 above)
Comment (TheJoin95) [reply src/ign-frontend/src/components/Demo.vue:265]: The modal asking for donation was visible after the 4th conversion.
With this change it'll be visible the first time then every 3rd conversion.

**Comment by TheJoin95** [comment]
> Some comments, and why this changes increase devex?

I drop the support on node-sass (which was using node-gyp) because outdated and not up to date anymore.
Basically it was impossible to run our application in the recent node version / python version.

Now it supports > node16 (running node20)

File: src/ign-frontend/src/components/Demo.vue (hunk 2 above)
Comment (Wabri) [reply src/ign-frontend/src/components/Demo.vue:265]: ok, make sense

**Review by Wabri — APPROVED** [review]
LGTM

**Pull request merged**